data/transactions/logic/fields.go

```diff
@@ -389,6 +389,41 @@ func (s gfNameSpecMap) getExtraFor(name string) (extra string) {
 	return
 }
 
+// EcDsaCurve is an enum for `ecdsa_` opcodes
+type EcDsaCurve int
```

Comment:
Same comment about the D, but it looks like this was very intentional, so if this how it's often broken up, fine by me.

Reply:
done